pkgs/applications/networking/twingate/default.nix

```diff
@@ -1,26 +1,27 @@
 { autoPatchelfHook
 , curl
 , dpkg
-, dbus_daemon
+, dbus
 , fetchurl
 , lib
 , libnl
-, libudev
+, udev
+, cryptsetup
 , stdenv
 , pkgs
 }:
 
 stdenv.mkDerivation rec {
   pname = "twingate";
-  version = "1.0.12";
+  version = "1.0.39.44791";	#add comment
```

Comment:
For other reviewers: we will change version here and URL below when we release linux client to master (to put prod values here)